docs/function-schemas.md

```diff
@@ -21,6 +23,7 @@
   * [Pretty Errors](#pretty-errors)
 * [Defn Schemas via metadata](#defn-schemas-via-metadata)
   * [TL;DR](#tldr)
+* [Polymorphic Functions](#polymorphic-schemas)
```

Comment:
This is also titled "Polymorphic Functions". Did you mean "Polymorphic Schemas"?